src/Connection.ts

```diff
@@ -123,14 +126,25 @@ export class Connection {
         });
     }
 
+    /**
+     * Close the connection. It can be re-opened using connect.
+     */
+    public close() {
+        this._closed = true;
+        this._autoReconnectHandler.stop();
+        this.stropheConnection.disconnect();
+        this.stropheConnection.reset();
+    }
+
     /**
      * Disconnect and complete the model observables.
-     * 
+     *
      * This Connection instance should not be used afterwards;
      * for a new connection, create a new instance.
-     * 
+     *
      */
     public disconnect() {
+        this._closed = true;
```

Comment:
Can't we call close() here? (in that case, just the 'reset' gets called that otherwise wouldn't get called)

Reply:
Yes, fixed!